### ISSUE TITLE
Install error (codept opam pkg) with OCaml 4.11.1

### ISSUE THREAD

**Issue by Fourchaux** [Octachron/codept#4]
With:
```  $ opam install codept ```
```
[ERROR] Package conflict!
  * No agreement on the version of ocaml:
    - (invariant) → ocaml-variants = 4.11.1+flambda → ocaml = 4.11.1
...
```

**Comment by Octachron** [comment]
The current opam package only works with <4.10 indeed.
I need to make a release with the master/0.11 branch.
Note that you can install with `opam pin add codept --dev-repo` and it should work fine.

**Comment by Octachron** [comment]
Codept 0.11.1 is on opam and support all OCaml from version 4.03.0 to the not yet released 4.13.0 .
